src/core/tests/symbol.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/core/symbol.hpp"
+
+#include <gtest/gtest.h>
+
+#include <memory>
+
+using namespace std;
+using namespace ov;
```

Comment:
```suggestion
namespace ov {
namespace test {
```
Consider wrap test in this namespace, then using `ov::` would not be required and I think `using std` is also not required.